bcgov/cas-registration
# 373 cement production

#373

This pr creates adds necessary configuration for Cement Production reporting activity

- feat: Add cement production reporting activity migration, data, and json
- chore: Tweak previous migration and add UISchema
- test: add test for cement production configuration

Steps to test:

- Make reset_db & make run
- Login as BCEID
- Nav to an SFO operation and set reporting activity to cement production
- Emission should have gas type as C02
- Field "Number of times in the reporting year that missing data procedures were followed" should exist
- Should have 5 methodologies and CEMS should have no reporting fields
